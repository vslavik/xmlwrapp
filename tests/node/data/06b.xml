<root><!--comment-->
<?piapp pitext?>
<chapter><title><![CDATA[text]]></title></chapter>
</root>
