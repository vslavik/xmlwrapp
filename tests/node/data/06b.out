type_element
    type_comment
    type_text
    type_pi
    type_text
    type_element
        type_element
            type_cdata
    type_text
